ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/execution/LedgerTimeAwareCommandExecutor.scala

```diff
@@ -70,7 +70,7 @@ final class LedgerTimeAwareCommandExecutor(
             contractStore
               .lookupMaximumLedgerTime(usedContractIds)
               .flatMap(maxUsedTime =>
-                if (!maxUsedTime.isAfter(commands.ledgerEffectiveTime)) {
+                if (!maxUsedTime.forAll(_.isAfter(commands.ledgerEffectiveTime))) {
```

Comment:
```suggestion
                if (!maxUsedTime.forall(_.isAfter(commands.ledgerEffectiveTime))) {
```
Fix compilation error.